examples/esplora_backend_asynchronous.rs

```diff
@@ -0,0 +1,91 @@
+use std::str::FromStr;
+
+use bdk::blockchain::Blockchain;
+use bdk::{
+    blockchain::esplora::EsploraBlockchain,
+    database::MemoryDatabase,
+    template::Bip84,
+    wallet::{export::FullyNodedExport, AddressIndex},
+    KeychainKind, SyncOptions, Wallet,
+};
+use bitcoin::{
+    util::bip32::{self, ExtendedPrivKey},
+    Network,
+};
+
+pub mod utils;
+
+use crate::utils::tx::build_signed_tx;
+
+use bdk_macros::await_or_block;
+
+/// This will create a wallet from an xpriv and get the balance by connecting to an Esplora server.
```

Comment:
Something like this.. 
```suggestion
/// This will create a wallet from an xpriv and get the balance by connecting to an Esplora server, using non blocking asynchronous calls with `reqwest`. 
```

Reply:
Done in e50fcb6c53964435d8a70f6fe396bf60bb75f1e2.